Fixed broken links in Nautobot README.md. (#4640)
nautobot/nautobot: README.md

The best platform for general feedback, assistance, and other discussion is our [GitHub discussions](https://github.com/nautobot/nautobot/discussions). To report a bug or request a specific feature, please open a GitHub issue using the [appropriate template](https://github.com/nautobot/nautobot/issues/new/choose).

> If you are interested in contributing to the development of Nautobot, please read our [contributing guide](https://docs.nautobot.com/projects/core/en/stable/development/) prior to beginning any work.

Reviewer: This should probably point to https://docs.nautobot.com/projects/core/en/stable/development/core/#contributing

Author: I feel like this should take the user to https://docs.nautobot.com/projects/core/en/stable/development/. Because they can be contributing via apps/core/jobs. Is this link restricted to core only?

Reviewer: The apps should have their own contributing guidelines that they post in their repositories. Since this lives in the Nautobot repo it should take you to the docs for contributing to Nautobot core.

## Related projects

> Please checkout [our nautobot topics](https://github.com/topics/nautobot) for a list of relevant community projects.

Reviewer 2: Perhaps?

```suggestion
Please check out [the GitHub `nautobot` topic](https://github.com/topics/nautobot) for a list of relevant community projects.
```

> Please checkout [our community repository](https://github.com/nautobot/community/blob/main/content/README.md) for a list of relevant community projects.

Reviewer 2: Possibly https://github.com/topics/nautobot instead?

> Please checkout [our community repository](https://github.com/nautobot/community/blob/main/content/README.md) for a list of relevant community projects.

Reviewer: Is this the correct link? It seems like this should go to a list of github projects, not blog posts and videos.
